src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -1096,4 +1103,16 @@ public static String escapeXpathValue(String value) {
 		return String.format("'%s'", value);
 	}
 
+	private boolean textContains(String actual, String text, boolean ignoreCase) {
```

Comment:
That is even better 👍 

One tiny idea: how about adding `static` modifier to these utility methods?